app/services/mixpanel_analytics.rb

```diff
@@ -0,0 +1,21 @@
+module MixpanelAnalytics
+  def requested_feed(owner:, repo:)
+    event = 'Fetched feed'
+    properties = { owner: owner, repo: repo }
+    track(event, properties)
+  end
+
+  private
+
+  def track(event, properties)
+    tracker.track(user_id, event, properties)
+  end
+
+  def tracker
+    @tracker ||= Mixpanel::Tracker.new(ENV['MIXPANEL_TOKEN'])
+  end
+
+  def user_id
+    request.remote_ip
```

Comment:
On further thought, this is sort of an abuse of the distinct_id that MP provides. It really is meant to mean a single person. IP may not be perfect, but it's what I have.